### DISCUSSION TITLE
why operator function for sm60/61 does nothing

### DISCUSSION THREAD

**Discussion by wzhcz8902** [NVIDIA/cutlass#1505] [Q&A] [answered]
https://github.com/NVIDIA/cutlass/blob/5c447dd84f8ae0e1d48ff9a2eae26ce8c4958101/include/cutlass/arch/simd_sm60.h#L48-L54

why the operator * does nothing?

**Comment by hwu36** [comment, marked as answer]
nothing in this file is implemented.  it is supposed to use [half2](https://docs.nvidia.com/cuda/cuda-math-api/group__CUDA__MATH____HALF2__ARITHMETIC.html) to implement these.
